rmw_connext_cpp/src/functions.cpp

```diff
@@ -176,7 +198,7 @@ rmw_create_publisher(
     return nullptr;
   }
 
-  auto node_info = static_cast<ConnextNodeInfo *>(node->data);
+  ConnextNodeInfo * node_info = static_cast<ConnextNodeInfo *>(node->data);
```

Comment:
I think I did that so my autocomplete would work, but I can change them back: 90d4c86